.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - "5"
+  - "4"
+  - "4.2"
+
+cache:
+  directories:
+    - node_modules
+
+before_script:
+  - NODE_ENV='ci'
+
+script:
+  - npm run test:ci
```

Comment:
I think I should iterate that this module is intended to add "default" rules to the consumers' project, and any of them can be overwritten.